File: app/src/main/java/com/jargle/elevationservice/elevationservice/ElevationService.java
package com.jargle.elevationservice.elevationservice;

import android.app.Service;
import android.content.Intent;
import android.os.IBinder;
import android.support.annotation.Nullable;

public final class ElevationService extends Service {

    public static final String TAG = "ElevationService";

    @Nullable
    @Override
    public IBinder onBind(Intent intent) {
        return null;
    }

    @Override
    public void onCreate() {
        super.onCreate();
    }
}

File: app/src/main/java/com/jargle/elevationservice/elevationservice/ElevationService.kt
package com.jargle.elevationservice.elevationservice

import android.app.Service
import android.content.Intent
import android.os.IBinder

class ElevationService : Service() {

    override fun onBind(p0: Intent?): IBinder? {
        return null
    }

    override fun onCreate() {
        super.onCreate()

    }
}
